lib/mayaUsd/fileio/jobs/readJob.h

```diff
@@ -128,9 +129,10 @@ class UsdMaya_ReadJob
     double _setTimeSampleMultiplierFrom(const double layerFPS);
 
     // Data
-    MDagModifier mDagModifierUndo;
-    bool         mDagModifierSeeded;
-    double       mTimeSampleMultiplier;
+    MDagModifier                    mDagModifierUndo;
+    bool                            mDagModifierSeeded;
+    double                          mTimeSampleMultiplier;
+    UsdMayaImportChaserRefPtrVector mImportChasers;
```

Comment:
Please document the purpose and lifetime of this member (don't hesitate to add documentation above as well).

Reply:
This is addressed in 50c5396.